core/src/main/scala/io/projectglow/transformers/LiftOverVariantsTransformer.scala

```diff
@@ -246,6 +246,15 @@ object LiftOverVariantsTransformer extends GlowLogging {
     val refStr = refSeq.getBaseString.substring(outputVc.getStart - 1, outputVc.getEnd)
 
     if (outputVc.getReference.getBaseString.toLowerCase != refStr.toLowerCase) {
+      if (outputVc.isBiallelic && outputVc.isSNP && refStr.equalsIgnoreCase(
```

Comment:
Why is this only for biallelic variants?

Reply:
This logic is pulled from `LiftoverVcf`, which only recovers swapping ref/alt alleles in biallelic SNPs: https://github.com/broadinstitute/picard/blob/a3021a7a7e00fcd5f05a9d6088c8f29a707d7420/src/main/java/picard/vcf/LiftoverVcf.java#L541